app/konnect/updates.md

```diff
@@ -104,19 +104,19 @@ Existing RBAC roles have been converted to [predefined teams](/konnect/org-manag
 
 : To set up a custom URL, see the [Dev Portal customization documentation](/konnect/dev-portal/customization) for more information.
 
-**Vitals metrics and dashboards**
-: The Vitals overview dashboard has been reworked for Plus and Enterprise tiers.
+**Analytics metrics and dashboards**
```

Comment:
I think we should leave these old release note updates as-is, and not change the name to "Vitals", since it was named "Vitals" at the time of the change. Instead, we need a new release note saying that Vitals has been renamed to Analytics.

Reply:
done